PaTTAN Room Displays/PaTTAN Room Displays/RoomDataService.cs

```diff
@@ -10,23 +10,40 @@ namespace PaTTAN_Room_Displays
 {
     internal class RoomDataService
     {
+        private DateTime _lastUpdated;
+        public DateTime lastUpdated {
```

Comment:
This is OK, but going forward I prefer to camelCase **private** and **protected** identifiers and to PascalCase **internal** and **public** identifiers (and to SCREAM_SNAKE constant identifiers)